### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jamesgk**
Lots of whitespace and variable name changes, but no functionality should have changed. Tests and benchmark results stayed the same AFAICT. I didn't added much in the way of comments.

One last nit: I don't really like having almost-identical functions `cubic_farthest_fit` and `_cubic_farthest_fit`. Removing `_cubic_farthest_fit` does add running time (~5-10%?) but personally I don't think it's worth it in this case.

### CODE REVIEW

**Comment by behdad** [comment]
> One last nit: I don't really like having almost-identical functions cubic_farthest_fit and _cubic_farthest_fit. Removing _cubic_farthest_fit does add running time (~5-10%?) but personally I don't think it's worth it in this case.

I have one more thing to optimize to use the `_cubic...` version, and gives another small boost.  So I'll add that and rename it to be more useful.

File: Lib/cu2qu/__init__.py [hunk 1]
```diff
@@ -177,9 +177,9 @@ def cubic_approx_spline(cubic, n, tolerance):
         if i == 1:
             q0, q1, q2 = spline[0], spline[1], (spline[1] + spline[2]) * .5
         elif i == n:
-            q0, q1, q2 = (spline[-3] + spline[-2]) * .5, spline[-2], spline[-1]
+            q0, q1, q2 = q2, spline[-2], spline[-1]
```
Comment (behdad) [inline Lib/cu2qu/__init__.py]: I was thinking about reusing q2.  It's nice, but belongs to separate commit.  Don't worry separating it though.

Comment (jamesgk) [inline Lib/cu2qu/__init__.py]: Yea I guess that's more than just a style change, my bad.
